network/network.go

```diff
@@ -110,7 +110,7 @@ func (n *Network) CheckHealth() map[string]core.Health {
 		}
 	}
 	// auth_config checks that the node is correctly configured to be authenticated by others
-	nodeDID, err := n.nodeDIDResolver.Resolve()
+	nodeDID, err := n.nodeDIDResolver.Resolve(context.TODO())
```

Comment:
This would be part of passing ctx through the health interface? Please link an issue if so.